src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -146,16 +159,20 @@ export interface IQuickInput {
 	show(): void;
 
 	hide(): void;
-
-	onDidHide: Event<void>;
-
-	dispose(): void;
 }
 
 export interface IQuickPick<T extends IQuickPickItem> extends IQuickInput {
 
 	value: string;
 
+	/**
+	 * A method that allows to massage the value used
+	 * for filtering, e.g, to remove certain parts.
+	 */
+	filterValue: (value: string) => string;
```

Comment:
@chrmarti this is a helper to massage the value used for filtering, which I need for quick access because each one has a prefix that I would like to exclude from filtering (e.g. `?@` should filter the help quick access down to `@`)